README.md

```
@@ -278,6 +278,7 @@ If you're accessing via a **custom domain** (e.g., `gmail.example.com`) instead
 - ✅ `http://localhost:8767/` (for local access)
 - ✅ `http://gmail.example.com:8767/` (custom domain)
 - ✅ `http://mygmail.duckdns.org:8767/` (dynamic DNS)
+- ✅ `https://gmail.example.com/` (HTTPS reverse proxy — requires [`OAUTH_REDIRECT_URI`](#3-for-https-with-reverse-proxy))
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)